### PR TITLE
fix-4274-upgrade-appwrite-to-1.0

### LINKED ISSUE #4274: 🪄 Upgrade ’demos-for-react’ to Appwrite 1.0
# 💭 Introduction

Appwrite is all about open source and community 👥 To emphasize this, we want to create **the best** developer experience possible by providing enough guides and examples for new Appwriters.

Your task is to upgrade our demo application **appwrite/demos-for-react** to the latest Appwrite version, which is 1.0. All of the functionality of the demo application must stay the same. 

### ✅ Task Summary

- [ ]  Ask to be assigned to the issue.
- [ ]  Wait to be assigned.
- [ ]  Upgrade appwrite/demos-for-react to support Appwrite 1.0.
- [ ]  Submit a pull request in **appwrite/demos-for-react**.

If you have questions, need any help, or just want to hang out, make sure to join us on our [Discord server](https://appwrite.io/discord).

**Happy Appwriting!**

### PR DESCRIPTION
**Pull request by omjikush09**
## What does this PR do?
upgraded to  appwrite to 1.0 (by upgrading sdk to 10.0.1). 
(Provide a description of what this PR does.)

## Test Plan
verified changes  work  by running React  servers and doing manual testing.
(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)
Issue
https://github.com/appwrite/appwrite/issues/4274

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

**Comment by Meldiron** [comment]
@omjikush09 , thanks for the PR! 🤯 Please give us some time to review it.

File: example-auth/src/utils.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
-appwrite
-  .setEndpoint('http://localhost/v1') // We set the endpoint, change this if your using another endpoint URL.
-  .setProject('ProjectID'); // Here replace 'ProjectID' with the project ID that you created in your appwrite installation.
+import {Client} from "appwrite"  // Import the appwrite library
 
-export { appwrite }; // Finally export the appwrite object to be used in projects.
+const client = new Client();
+ 
+client
+  .setEndpoint('http://localhost:8000/v1') // We set the endpoint, change this if your using another endpoint URL.
```
Comment (codercatdev) [inline example-auth/src/utils.js:6]: Let's set this back the the generics `http://localhost/v1` and `ProjectId`.

File: example-cropping/src/App.js [hunk 2]
```diff
@@ -10,23 +11,24 @@ import SignUp from './components/SignUp';
 
 function App() {
   const [appwrite] = useState(
-    appWrite({ endpoint: 'http://localhost:4000/v1', projectId: '5f830bed0e5bf' }),
+    appWrite({ endpoint: 'http://localhost:8000/v1', projectId: '632354c21d497895d2dd' }),
```
Comment (codercatdev) [inline example-cropping/src/App.js]: Can we pull in from utils?

File: example-cropping/src/components/SignUp.jsx [hunk 3]
```diff
@@ -44,7 +45,8 @@ export default function SignUp({ currentPage, appwrite, setCurrentPage }) {
       return;
     }
     try {
-      await appwrite.account.create(email, password);
+      const account = new Account(appwrite)
+      await account.create("unique()",email, password);
```
Comment (codercatdev) [inline example-cropping/src/components/SignUp.jsx]: You can now import `ID` from `appwrite` for helpers like unique.
Then update like below
```
      await account.create(ID.unique(),email, password);
```

File: example-crud/src/actions/index.js [hunk 4]
```diff
@@ -29,11 +33,11 @@ const logout = async history => {
 
 const creatUserDocument = async userData => {
   try {
-    return appwrite.database.createDocument(
+    return database.createDocument(
+      databaseID,
       collectionID,
+      "unique()",
```
Comment (codercatdev) [inline example-crud/src/actions/index.js]: Please pull from ID class, see above comment.

File: example-crud/src/actions/index.js [hunk 5]
```diff
@@ -29,11 +33,11 @@ const logout = async history => {
 
 const creatUserDocument = async userData => {
```
Comment (codercatdev) [inline example-crud/src/actions/index.js:32]: Could we add to the README some basic instructions for creating the correct collections and structure with the new project.

File: example-locale/src/components/YourInfo.js [hunk 6]
```diff
@@ -7,6 +7,7 @@ const YourInfo = () => {
   // Promise to get the location details from the Locale API.
   response.then(
     function (res) {
+      console.log(res);
```
Comment (codercatdev) [inline example-locale/src/components/YourInfo.js:10]: Should we leave this?

**Review by codercatdev — CHANGES_REQUESTED** [review]
@omjikush09 can you please see the comments on the review and fix it? 

Thank you so much for this!!

**Comment by omjikush09** [comment]
Done with all requested changes and tested everyProject

**Review by codercatdev — APPROVED** [review]
Thank you!

**Comment by codercatdev** [comment]
@appwrite/core looks ready to merge

**Comment by Meldiron** [comment]
Thank you so much for the PR 🤩. We're adding the `hacktoberfest-accepted` label to ensure this PR counts towards your Hacktoberfest contributions count. With that said, please stay active on this PR to address any comments once you receive a review. Happy Hacktoberfest! 🎃

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Pull request merged**